Monika After Story/game/script-topics.rpy

```diff
@@ -13046,3 +13046,158 @@ label mas_show_unseen:
     m 1esa "Just give me a second.{w=0.5}.{w=0.5}.{nw}"
     m 3hua "There you go!"
     return
+    
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_installed_python",
+            category=['misc'],
+            prompt="I installed Python",
+            pool=True,
+            unlocked=True
+        )
+    )
+label monika_assume_code_level:
+    m 1etc "Have I been underestimating your coding skills?{nw}"
+    $ history_list.pop()
+    menu:
+        m 1etc "Have I been underestimating your coding skills?{fast}"
+        "Yes":
+            m 1hksdlb "Ahaha, I'm sorry, [player]!"
+            m 3rka "I didn't mean to- I just never thought to ask."
+        "No":
+            m 1ekb "Okay, good."
+            m 3eka "I just wanted to make sure I wasn't assuming your skill level.
+            
+label monika_python_version:
+    m 1eub "What version of Python did you download?{fast}"
+    $ _history_list.pop()
+    menu:
+        m 1eub "What version of Python did you download?{fast}"
+        #python 2 and python 3, need to figure out which one is used for MAS.
```

Comment:
She actually did specify that the mod uses Python2 in the "What is Python?" Python tip. _"Since this mod runs on a Ren'Py version that runs on Python2, I won't be talking about Python3 too often._

Reply:
Eep! Thank you!